internal/controller/postgrescluster/pgmonitor.go

```diff
@@ -221,25 +221,38 @@ func (r *Reconciler) reconcileMonitoringSecret(
 
 	intent.Data = make(map[string][]byte)
 
-	if len(existing.Data["password"]) == 0 || len(existing.Data["verifier"]) == 0 {
+	// Copy existing password and verifier into the intent
+	password, set := existing.Data["password"]
+	if set {
+		intent.Data["password"] = password
+		verifier, set := existing.Data["verifier"]
+		if set {
+			intent.Data["verifier"] = verifier
+		}
+	}
+
+	// When password is unset, generate a new one
+	// TODO (jmckulk): Consider password types (i.e., AlphaNumeric)
```

Comment:
📝 As I recall, AlphaNumeric was added to accommodate tools that are difficult to configure with `!` and `$` and similar.

Passwords with symbols are stronger/better (kinda) than those without, so I prefer we use them when possible.

Reply:
So we would never want to generate an Alphanumeric password? Meaning we can remove this TODO

I know user passwords have a user requirement for Alphanumeric, but this password isn't something that any apps or users should interact with. Only the exporter should be connecting through ccp_monitoring

Reply:
I removed this comment - users shouldn't be using this password, so I think we can use whatever type we think is appropriate